scripts/extract-l10n-merge.sh

```diff
@@ -0,0 +1,64 @@
+#! /bin/bash
```

Comment:
Curious, is it not explained in the context section of the PR description? The reason I ask is since we are discussing how relevant that section is recently and I'd like to ask if it answers your question before answering it here.

If that section does not answer, can you tell me what is still unclear?

Reply:
I was expected the existing script to be modified to allow only extraction, or a new script doing only extraction to be made and called from the existing script, instead of having the 2 live in parallel.

Reply:
I can update the existing script to call the new script, but according to the plan I laid out in the ticket, the existing script will eventually be replaced entirely by the new script plus the outer CI. In the end all we will need in "the script" is the extraction/merge bit.

But if you prefer I can update the existing script to call this script and remove that code. I don't have a strong opinion on that.

Reply:
Added the change. In addition I added a DRY_MODE flag for the extract script. I was needing to test it locally to make sure everything works, but it kept changing branches and it seemed like a pretty valid use case to want to run that script without committing anything, which is essentially the same as running the new script.

Like I said, if all goes to plan, the existing script will be deleted, but for now it is more easily executed without consequence.